docs/csharp/basic-types.md

```diff
@@ -2,6 +2,7 @@
 title: Basic Types - C# Guide
 description: Learn about the core types (numerics, strings, and object) in all C# programs
 ms.date: 10/10/2016
+ms.technology: csharp-fundamentals
```

Comment:
this one seems to be the most common under the csharp folder. add that to docfx json perhaps like the following and then this only has to be applied to the ones that are different?

```json
 "docs/csharp/**.md": "csharp-fundamentals",
```

Reply:
I like this thought, but I'm going to leave the tags in these files. As part of the reorg and new TOC, I'll be moving these to more appropriate locations. The `docs/csharp` folder won't have articles in it. This way, I have one less place to look as the new folder structure emerges.

Reply:
Makes sense!